feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -31,7 +31,14 @@ export class ConfiguratorPriceComponent {
     if (this.formula.priceTotal) {
       return this.priceTotal;
     } else {
-      return '+ ' + this.formula.price?.formattedValue;
+      if (this.formula.price?.value) {
+        if (this.formula.price?.value > 0) {
+          return '+' + this.formula.price?.formattedValue;
+        } else if (this.formula.price?.value < 0) {
+          return this.formula.price?.formattedValue || '';
```

Comment:
```suggestion
          return this.formula.price?.formattedValue ?? '';
```
please use the  [Nullish Coalescing Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) to set defaults instead of logical OR, which can be cumbersome.

Reply:
Absolutely agree! Done!